addons/calendar/tests/test_attendees.py

```diff
@@ -68,7 +68,7 @@ def test_attendee_added_multi(self):
             'stop': datetime(2019, 10, 27, 18, 0),
         })
         events = self.event | event
-        events.partner_ids = self.partner
+        events.sudo().partner_ids = self.partner
```

Comment:
Shouldn't user_id be populated with current user by default, hence having write access on it ?